### ISSUE TITLE
BUG Android+Talkback with simple combobox

### ISSUE THREAD

**Issue by jdeyla** [w3c/aria-practices#2855]
Hi, 
we have tested this example
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/
with many Android (12, 13, 14)   devices + Chrome and **the example does not work:**

it is not possible to make any choice (strange because listbox and options should be supported by chrome+takback)

**Comment by TrevorRice** [comment]
We've noticed the same with a select-only combobox of our own following a similar approach to APG's. Here's a potentially related issue: https://issuetracker.google.com/issues/274660397

**Comment by KaiWongTDOC** [comment]
Hi, this is still an issue as of June 13, 2025 and impacting many combo boxes across websites. 

Description: Cannot expand into the combo box in mobile web chrome browser using android mobile device. 

**Expected Result:**  User should be able to navigate into the Expanded list when using screen reader

**Actual Result:**  User cannot navigate into the Expanded List when using screen reader

**Device(s)/OS:** Samsung Galaxy

**Browser(s):** Chrome 130, Safari 17.4.1, Mobile Chrome, Mobile Safari 

**Screen Reader(s):**  Talkback, Voiceover
